src/messages.rs

```diff
@@ -75,6 +75,11 @@ pub struct BuildScript {
     pub cfgs: Vec<PathBuf>,
     /// The environment variables to add to the compilation
     pub env: Vec<(String, String)>,
+    /// The `OUT_DIR` environment variable where this script places its output
+    ///
+    /// Added in Rust 1.41.
+    #[serde(default)]
```

Comment:
This caused a runtime panic in rust-analyzer(nothing to problematic, just a minor annoyance :) ) I guess this is an argument for using options in similar cases in the future.

https://github.com/rust-analyzer/rust-analyzer/issues/5125

Reply:
oh.. I guess just using `default` for older compilers isn't making this backwards compatible. Thanks, I'll keep that in mind for the future.